### PR TITLE
feat(website): Document Homebrew specific version

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

The convention in Homebrew is to use `@<version>` without the `v`, so this required a bit more changes.
Instead of adding to the existing code block, we can also use a callout for it to get more visibility (I'm not sure we want that)

### CODE REVIEW

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Why do we need full and trimmed? can we have just one?

**Comment by erezrokah** [comment]
> Why do we need full and trimmed? can we have just one?

We use `v` in our tags https://github.com/cloudquery/cloudquery/releases/tag/cli-v2.0.25, and the naming convention in Homebrew is to not have `v`, so if we need to chose I think we'll need to change our tags

**Comment by erezrokah** [comment]
We can also do something like `vVERSION_CLI` in the docs

File: website/components/mdx/_download-linux.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Tabs, Tab } from '../Tabs'
 <Tab>
 
 ```bash copy
-curl -L https://github.com/cloudquery/cloudquery/releases/download/VERSION_CLI/cloudquery_linux_amd64 -o cloudquery
```
Comment (erezrokah) [inline website/components/mdx/_download-linux.mdx:8]: Not having `cli-` prefix here is a bug also on https://www.cloudquery.io/

Comment (erezrokah) [reply website/components/mdx/_download-linux.mdx:8]: Extracted this to a separate PR https://github.com/cloudquery/cloudquery/pull/5484

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**